### PR TITLE
Update endpoint-security-edr-policy.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated Step 6 per issue#https://github.com/MicrosoftDocs/memdocs/issues/2913

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by PRMerger3** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by yogkumgit** [comment]
@Brenduns Please review and merge this Pr. Thanks

File: memdocs/intune/protect/endpoint-security-edr-policy.md [hunk 1]
```diff
@@ -173,7 +173,7 @@ Before you can deploy policy to devices managed by Configuration Manager, set up
 
 5. On the **Basics** page, enter a name and description for the profile, then choose **Next**.
 
-6. On the **Configuration settings** page, configure the settings you want to manage with this profile. The onboarding package is automatically included and isn’t something you can configure.
+6. On the **Configuration settings** page, Choose **Auto from Connector**  for **Microsoft Defender for Endpoint Clinet configuration package type**. Configure the **Sample Sharing** and **Telemetry Reporting Frequency** settings you want to manage with this profile.
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/endpoint-security-edr-policy.md:176]:
```suggestion
6. On the **Configuration settings** page, Choose **Auto from Connector** for **Microsoft Defender for Endpoint Client configuration package type**. Configure the **Sample Sharing** and **Telemetry Reporting Frequency** settings you want to manage with this profile. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Comment by yogkumgit** [comment]
@msbemba gentle reminder

**Pull request merged**